### PR TITLE
Central Package Management

### PR DESCRIPTION
**Pull request by williambohrmann3**
# Description

Runtime package references should be using the latest daily build in our development branch. This will be accomplished through central package management, so only one place needs to be updated with our daily build.

## Type of change

- Other enhancement

## Platforms tested on

- [x] WPF .NET 6
- [x] WPF Framework
- [x] WinUI
- [x] MAUI WinUI
- [ ] MAUI Android
- [x] MAUI iOS
- [ ] MAUI MacCatalyst

## Checklist

- [x] Self-review of changes
- [x] All changes work as expected on all affected platforms
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files

### CODE REVIEW

**Comment by williambohrmann3** [comment]
We'll be moving to [Central Package Management](https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management)

File: src/WinUI/ArcGIS.WinUI.Viewer/ArcGIS.WinUI.Viewer.csproj [hunk 1]
```diff
@@ -79,10 +79,6 @@
     </Compile>
   </ItemGroup>
 
-  <ItemGroup>
```
Comment (williambohrmann3) [inline src/WinUI/ArcGIS.WinUI.Viewer/ArcGIS.WinUI.Viewer.csproj:82]: Removing this, as sketch-on-map samples has been replaced with geometry editor sample.

File: src/Directory.Packages.props [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <ArcGISMapsSDKVersion>200.2.0</ArcGISMapsSDKVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Esri.ArcGISRuntime" Version="$(ArcGISMapsSDKVersion)" />
+	<PackageVersion Include="Esri.ArcGISRuntime.Toolkit.WPF" Version="$(ArcGISMapsSDKVersion)" />
```
Comment (dotMorten) [inline src/Directory.Packages.props:8]: tabs and spaces mixed

File: src/Directory.Packages.props [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <PackageVersion Include="System.Speech" Version="7.0.0" />
+	<PackageVersion Include="System.Drawing.Common" Version="7.0.0" />
+    <PackageVersion Include="System.Security.Cryptography.ProtectedData" Version="7.0.0" />
+    <PackageVersion Include="CommunityToolkit.WinUI.UI.Controls" Version="7.1.2" />
+    <PackageVersion Include="CommunityToolkit.WinUI.UI.Controls.Markdown" Version="7.1.2" />
+	<PackageVersion Include="CommunityToolkit.Maui" Version="5.0.0" />
+    <PackageVersion Include="CommunityToolkit.Mvvm" Version="8.1.0" />
+	<PackageVersion Include="WinUIEx" Version="1.8.0" />
+    <PackageVersion Include="System.Security.Cryptography.ProtectedData" Version="7.0.0" />
+	<PackageVersion Include="Xamarin.AndroidX.AppCompat" Version="1.5.1.1" />
```
Comment (dotMorten) [inline src/Directory.Packages.props:26]: Do we need to centrally manage packages that are only used in a single project? (goes for several here)

File: src/WPF/WPF.Viewer/ArcGIS.WPF.Viewer.Net.csproj [hunk 4]
```diff
@@ -60,12 +60,12 @@
...
-    <PackageReference Include="Esri.ArcGISRuntime.Toolkit.WPF" Version="200.2.0" />
-    <PackageReference Include="Esri.ArcGISRuntime.WPF" Version="200.2.0" />
-    <PackageReference Include="Markdig" Version="0.30.4" />
-    <PackageReference Include="System.Speech" Version="7.0.0" />
+    <PackageReference Include="Esri.ArcGISRuntime.Hydrography" />
+    <PackageReference Include="Esri.ArcGISRuntime.LocalServices" />
+    <PackageReference Include="Esri.ArcGISRuntime.Toolkit.WPF" />
+    <PackageReference Include="Esri.ArcGISRuntime.WPF" />
+    <PackageReference Include="Markdig" />
+	<PackageReference Include="System.Speech" />
```
Comment (dotMorten) [inline src/WPF/WPF.Viewer/ArcGIS.WPF.Viewer.Net.csproj:68]: indent

File: src/WPF/WPF.Viewer/ArcGIS.WPF.Viewer.NetFramework.csproj [hunk 5]
```diff
@@ -282,14 +282,12 @@
...
-    <PackageReference Include="Markdig" Version="0.30.2" />
-    <PackageReference Include="System.Text.Json">
-      <Version>7.0.1</Version>
-    </PackageReference>
+    <PackageReference Include="Esri.ArcGISRuntime.Hydrography" />
+    <PackageReference Include="Esri.ArcGISRuntime.LocalServices" />
+    <PackageReference Include="Esri.ArcGISRuntime.Toolkit.WPF" />
+    <PackageReference Include="Esri.ArcGISRuntime.WPF" />
+    <PackageReference Include="System.Text.Json"/>
+	<PackageReference Include="Markdig" VersionOverride="0.30.2"/>
```
Comment (dotMorten) [inline src/WPF/WPF.Viewer/ArcGIS.WPF.Viewer.NetFramework.csproj:290]: 0.30.4 doesn't work here?

Comment (williambohrmann3) [reply src/WPF/WPF.Viewer/ArcGIS.WPF.Viewer.NetFramework.csproj:290]: I think we can upgrade - markdown rendering is working when looking at sample description and viewer settings.

**Review by dotMorten — APPROVED** [review]

**Pull request merged**